mender/values.yaml

```
@@ -16,6 +16,7 @@ global:
     AWS_SECRET_ACCESS_KEY: mysecretkey
     AWS_FORCE_PATH_STYLE: "true"
     AWS_TAG_ARTIFACT: "true"
+    useServiceAccount: "false"
```

To keep the configuration option consistent with the other ones, I'd rather suggest:
```suggestion
    AWS_SERVICE_ACCOUNT_NAME: ""
```